### DISCUSSION TITLE
[New Feature] Negative Prerequisites

### DISCUSSION THREAD

**Discussion by MNOrac** [Vinifera-Developers/Vinifera#1438] [Feature Requests]
### Description:
Sometimes it's useful to be able to make something stop being buildable when certain criteria are met. A good example is a situation where a player captures an interesting enemy or tech building that grants access to new construction options. Negative Prerequisites would allow substitutions instead of additions.

### Possible Implementation:
Ares implements this as documented here:
https://ares-developers.github.io/Ares-docs/new/prerequisites.html#negative-prerequisites

**Comment by Longshanks1** [comment]
Would be a useful feature if possible
